tests/suites/test_suite_mpi.function

```diff
@@ -1720,6 +1720,378 @@ exit:
 }
 /* END_CASE */
 
+/* BEGIN_CASE */
+void mbedtls_mpi_core_add_if( char * input_l, char * input_r,
```

Comment:
When I see `mbedtls_mpi_core_add_if`, my brain goes to “library function”, not “test function”, and I really wish we stopped having the `mbedtls_` prefix on test functions. There's a precedent for using `mpi_foo` and not `mbedtls_mpi_foo` in this test suite already.

Yes I know we've already disagreed about this. My brain still takes the wrong path after all these years looking at the test suites that use the `mbedtls_xxx` test function naming convention.

Reply:
I've renamed the test cases. Now my brain recoils at the lack of prefix in the `.function` file :)

I agree that initially I found the identical names confusing, but my brain quickly adapted to `.c` vs `.function`, and (as I've said) I found it comforting (bizarre, but that's the right word).

Maybe we should have the "function" in the `.function` file (in quotes, since the build system mangles it's name) required to start with `test_`, and the lines in `.data` files NOT start with `test_`. So then the line in the `.data` file would look like

```
mbedtls_mpi_core_add_if:"1":"10000":"10001":0:"10001":0
```

and the `.function` file would have

```
void test_mbedtls_mpi_core_add_if( char * input_A, char * input_B,
                                   char * input_X4, int carry4,
                                   char * input_X8, int carry8 ) 
```